### DISCUSSION TITLE
The unknown tags worry me. There are dozens of quasi-proprietary extensions to tiff where manufacturers invent private tags to hold parts of the image. Hopefully this isn't that.

### DISCUSSION THREAD

**Discussion by colinholbrook** [libvips/libvips#2584] [Q&A]
The unknown tags worry me. There are dozens of quasi-proprietary extensions to tiff where manufacturers invent private tags to hold parts of the image. Hopefully this isn't that.

_Originally posted by @jcupitt in https://github.com/libvips/ruby-vips/issues/112#issuecomment-308025383_

I see that the issue of unknown ImageJ TIFF tags was a problem back in 2017, and remains an issue to this day in libvips 8.12.  

Opening TIFF files saved with recent versions of FIJI produces this warning from libvips:
(process:940): VIPS-WARNING **: 12:27:40.350: Unknown field with tag 50838 (0xc696) encountered
(process:940): VIPS-WARNING **: 12:27:40.351: Unknown field with tag 50839 (0xc697) encountered

These two tags correspond to:
    50838: ("ImageJMetaDataByteCounts", LONG, 0),
    50839: ("ImageJMetaData", UNDEFINED, 1),
(see https://pillow.readthedocs.io/en/stable/_modules/PIL/TiffTags.html)

It would be helpful if vips could open images with these tags without raising warnings.  They apparently contain metadata and not image content.

**Comment by jcupitt** [comment]
Hi @colinholbrook,

The warnings are from libtiff, not libvips, although libvips is reporting them.

You can handle or suppress libvips warnings in various ways, eg. set the `VIPS_WARNING` env var to hide all warnings.
